java/build-android.gradle

```diff
@@ -24,7 +26,11 @@ project.buildDir = buildDir
 project.version = rootProject.file('../VERSION_NUMBER').text.trim()
 project.group = "com.microsoft.onnxruntime"
 
-def mavenArtifactId = project.name + '-mobile'
+def mavenArtifactId = isMobileBuild ? project.name + '-mobile' : project.name
```

Comment:
Also I am not exactly sure if the mavenArtifactId `onnxruntime` will be in conflict with this https://mvnrepository.com/artifact/com.microsoft.onnxruntime/onnxruntime, probably we will need to have a different mavenArtifactId

Reply:
> Also I am not exactly sure if the mavenArtifactId `onnxruntime` will be in conflict with this https://mvnrepository.com/artifact/com.microsoft.onnxruntime/onnxruntime, probably we will need to have a different mavenArtifactId
> 
> If we have a different artifact id, then this line needs update
> 
> https://github.com/microsoft/onnxruntime/blob/b68b62f36c95c9ee6fcf482fa78c6f1a2df1b6f9/tools/ci_build/github/azure-pipelines/templates/android-java-api-aar.yml#L42
> 
> and here
> 
> https://github.com/microsoft/onnxruntime/blob/e999a9a871bf3a27040b0ccadf02b12a2c0f23ba/tools/ci_build/github/azure-pipelines/templates/android-java-api-aar-test.yml#L16

These 2 lines may need be updated to `default: 'onnxruntime-android'`

Reply:
What about https://github.com/microsoft/onnxruntime/blob/e999a9a871bf3a27040b0ccadf02b12a2c0f23ba/tools/ci_build/github/azure-pipelines/templates/android-java-api-aar-test.yml#L58?

Should it always be copying to onnxruntime-mobile.aar?

Reply:
The `onnxruntime-mobile.aar` is hardcoded in the test project here

https://github.com/microsoft/onnxruntime/blob/f9b6eef05f58c5780ca31ed76a55aa5df1f488b0/java/src/test/android/app/build.gradle#L47

Since we generate aar packages with different names, it will be copied to the test project to `onnxruntime-mobile.aar` to avoid modifying the `build.gradle` every time we run test, the actual name in the test project does not matter, can even use `chuck-norris.aar` or `darth-vader.aar`